en/material-system/overview.md

```diff
@@ -10,7 +10,7 @@ The general structure of the system is as follows:
 `EffectAsset` is an shading procedure description file, written by both engine and game developers.<br>
```

Comment:
`EffectAsset` is an shading procedure description file, written by both engine and game developers.<br>
-->
`EffectAsset` is a shading procedure description file, written by both engine and game developers.<br>